motoman_driver/include/motoman_driver/io_relay.h

```diff
@@ -0,0 +1,82 @@
+﻿/*
+ * Software License Agreement (BSD License)
+ *
+ * Copyright (c) 2013, Southwest Research Institute

```

Comment:
I would suggest to update the copyright here.

Reply:
Addressed in 8c1757d